build.gradle

```diff
@@ -243,10 +243,13 @@ project('serializers:avro') {
     dependencies {
         compile project(':serializers:shared')
         compile group: 'org.apache.avro', name: 'avro', version: avroVersion
+        compileOnly group: 'io.pravega', name: 'pravega-client', version: pravegaVersion
```

Comment:
Could we avoid this dependency? Having the serializers require Pravega client coupled it to Pravega client, which doesn;'t look right to me. What do you say?

Reply:
I am not sure i follow your suggestion. However, i have also created a separate PR for build.gradle changes in #160 .
let me explain the intent for this change
schema registry serializers depend on pravega Serializer interface which they provide an implementation for. So they take a dependency on rpavega client. 
For Serializers we have a "serializers:shared" project which had compile dependency on pravega-client and serializers:avro etc took dependency on serializers:shared. 
now the idea is to make pravega client dependency compile only so we dont bundle pravega client with schema registry serializers. 
so i changed it in serializers:shared and since pravega client is only compileonly dependency in serializers:shared, we have to make the same change in all serializers (protobuf, avro, json).